docs/src/Callout.doc.js

```diff
@@ -22,168 +47,346 @@ card(
         type: 'string',
         required: true,
         defaultValue: null,
-        description: [
-          'Text to render inside the callout to convey detailed information to the user. The message text has a fixed size.',
-        ],
-        href: '',
+        description: `
+          Main content of Callout. Content should be localized.
+
+          See [Best Practices](#Best-practices) for more info.`,
       },
       {
         name: 'dismissButton',
         type: '{| accessibilityLabel: string, onDismiss: () => void, |}',
-        required: false,
         defaultValue: null,
-        description: [
-          'Callback fired when the dismiss button is clicked (pressed and released) with a mouse or keyboard.',
-          'Supply a short, descriptive label for screen-readers to provide sufficient context about the dismiss button action. IconButtons do not render text for screen readers to read requiring an accessibility label.',
-          'Accessibility: `accessibilityLabel` populates aria-label.',
-        ],
-        href: '',
+        description: `
+          When specified, adds a dismiss button to Callout. See the [Dismissible variant](#Dismissible) for more info.
```

Comment:
I wonder if `When specified, ` is redundant as `Adds` conveys the same optional/additional nature.